### PR TITLE
Connect dashboard statistics

### PR DESCRIPTION
**Pull request by AaronAlemi**
Merging in order to receive fix for an error with the teams table possibly causing issues with rendering the charts on initial load. Will resolve the merge conflicts soon after making this PR

### CODE REVIEW

File: bugtracker-ui/src/components/Dashboard/Dashboard.jsx [hunk 1]
```diff
@@ -79,15 +91,32 @@ export default function Dashboard() {
     //setIsLoading(false) 
   }
 
+
   window.onload = function () {
     //getTeamsTable()
+    console.log("Inside window.onload")
```
Comment (LifeBringer) [inline bugtracker-ui/src/components/Dashboard/Dashboard.jsx:97]: Please delete or comment out.

File: bugtracker-ui/src/components/Dashboard/Dashboard.jsx [hunk 2]
```diff
@@ -79,15 +91,32 @@ export default function Dashboard() {
...
+    fetchDashboardStatistics()
+    console.log("Fetched dashboard statistics onload complete")
```
Comment (LifeBringer) [inline bugtracker-ui/src/components/Dashboard/Dashboard.jsx:99]: Please delete or comment out.

File: bugtracker-ui/src/components/Dashboard/Dashboard.jsx [hunk 3]
```diff
@@ -79,15 +91,32 @@ export default function Dashboard() {
...
   }
 
   useEffect(() => {
      //clearTeams()
-     renderCharts()
-     fetchProjects()
-     newFetchTeamsTableData(teams)
+
+     fetchDashboardStatistics()
+     console.log("dashboard statistics:", dashboardStatistics)
```
Comment (LifeBringer) [inline bugtracker-ui/src/components/Dashboard/Dashboard.jsx:107]: Please delete or comment out.

File: bugtracker-ui/src/contexts/statistics.jsx [hunk 4]
```diff
@@ -0,0 +1,43 @@
...
+  const [dashboardStatistics, setDashboardStatistics] = useState({})
+  const [isLoading, setIsLoading] = useState(false);
+  const [error, setError] = useState("");
+
+  const fetchDashboardStatistics = async () => {
+    setIsLoading(true);
+    let stats = await apiClient.getAllStatistics()
+    console.log("fetchDashboardStatistics stats:", stats)
+    setDashboardStatistics(stats)
+    console.log("fetchDashboardStatistics dashboardStatistics:", dashboardStatistics)
```
Comment (LifeBringer) [inline bugtracker-ui/src/contexts/statistics.jsx:17]: Please delete or comment out. x 2

File: bugtracker-ui/src/contexts/statistics.jsx [hunk 5]
```diff
@@ -0,0 +1,43 @@
...
+    setIsLoading(false);
+  };
+
+  useEffect(() => {
+    console.log("Enter statistics useEffect")
```
Comment (LifeBringer) [inline bugtracker-ui/src/contexts/statistics.jsx:22]: Please delete or comment out.

File: bugtracker-ui/src/services/charts.js [hunk 6]
```diff
@@ -4,24 +4,87 @@
 */
 
 
-export default function renderCharts() { 
+
+
+export default function renderCharts(dashboardStatistics) { 
+
+    
+    console.log("RenderCharts dashboardStatistics:", dashboardStatistics)
```
Comment (LifeBringer) [inline bugtracker-ui/src/services/charts.js:12]: Please delete or comment out.

File: bugtracker-ui/src/services/charts.js [hunk 7]
```diff
@@ -4,24 +4,87 @@
...
+        statusData[2] = parseInt(statusType.totaltickets)
+      }
+      else if (statusType.status === 'submitted') {
+        statusData[3] = parseInt(statusType.totaltickets)
+      }
+      else if (statusType.status === 'resolved')   {
+        statusData[4] = parseInt(statusType.totaltickets)
+      }
+    })
+    console.log("statusData: ", statusData)
```
Comment (LifeBringer) [inline bugtracker-ui/src/services/charts.js:39]: Please delete or comment out.

**Comment by AaronAlemi** [comment]
Changes made

**Pull request merged**